.github/workflows/build-deploy.yml

```
@@ -0,0 +1,170 @@
+    runs-on: ubuntu-latest      
+    env:
+      GO_VERSION: "1.21"
+      PROJECT_NAME: "⚡ BitcoinDeepaBot - Prod"
+      DEPLOY_PATH: "/root/projects/bitcoin/deepa-bot/prod/"
+      PROJECT_DIR: "bot"
+      SERVICE_NAME: "bitcoin_deepa_bot_prod.service"
+      BINARY_NAME: "BitcoinDeepaBot"
+    
+    steps:
+      - name: Notify Build Start
+        run: |          
```

actionlint (1.7.7)

"github.event.head_commit.message" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions for more details

(expression)